lib/validators/liveValidator.js

```diff
@@ -391,6 +392,11 @@ class LiveValidator {
     }
     let request = requestResponseObj.liveRequest;
     let response = requestResponseObj.liveResponse;
+
+    if (http.STATUS_CODES[response.statusCode] === undefined && utils.statusCodeStringToStatusCode[response.statusCode] !== undefined) {
```

Comment:
- Adding a comment would help in the future
- rather than explicitly checking for undefined, in this scenario it is better to check for a `falsey` value. what if the value is `null`, then the check will fail. Because this is our stupid javascript, it is better to check the presence of parent before accessing the nested property `response.statusCode`.

```javascript
// Added support to handle status code string (ex: "OK") along the status code number (ex: 200). 
if (response && !http.STATUS_CODES[response.statusCode] && utils.statusCodeStringToStatusCode[response.statusCode]) {
  response.statusCode = utils.statusCodeStringToStatusCode[response.statusCode]; 
}
```

Reply:
should be undefined when checking in the dictionary, but at least is shorter in this case.